docs/reference/migration/migrate_7_0/api.asciidoc

```diff
@@ -22,3 +22,6 @@ The following parameters starting with underscore have been removed:
 Instead of these removed parameters, use their non camel case equivalents without
 starting underscore, e.g. use `version_type` instead of `_version_type` or `versionType`.
 
+
+==== The parameter `fields` deprecated in 6.x has been removed from Bulk request 
+and Update request.
```

Comment:
Can you also add a note about the change you update request?

I'm not *sure* this is the right file for them but you may as well put them here for now and we'll move them if we decide there is a better spot.